app/main.py

```diff
@@ -2,0 +1,28 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> None:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+    def hide(self) -> bool:
```

Comment:
The `hide` method is defined to return a `bool`, but it currently does not return anything. Consider returning the new state of `self.hidden`.